File: C#/BluffinMuffin.Protocol.DataTypes/ConfigurableWaitingTimes.cs
using BluffinMuffin.Protocol.DataTypes.Attributes;

namespace BluffinMuffin.Protocol.DataTypes
{
    /// <summary>
    /// The waiting times (At different stage of the game, the server will wait before continuing to making it feel real !)
    /// </summary>
    public class ConfigurableWaitingTimes
    {
        /// <summary>
        /// Amount of miliseconds (ms) to wait after a player do something.
        /// </summary>
        [ExampleValue(500)]
        public int AfterPlayerAction { get; set; }
        
        /// <summary>
        /// Amount of miliseconds (ms) to wait after cards are dealed
        /// </summary>
        [ExampleValue(500)]
        public int AfterBoardDealed { get; set; }
        /// <summary>
        /// Amount of miliseconds (ms) to wait after pot is won
        /// </summary>
        [ExampleValue(2500)]
        public int AfterPotWon { get; set; }
    }
}

File: Python/BluffinMuffin.Protocol/DataTypes/ConfigurableWaitingTimes.py
from DataTypes.Enums.GameTypeEnum import GameTypeEnum

__author__ = 'ericm'
_project_ = 'BluffinMuffin.Protocol'


class ConfigurableWaitingTimes:
    def __init__(self, obj):
        self.after_player_action = obj['AfterPlayerAction']
        self.after_board_dealed = obj['AfterBoardDealed']
        self.after_pot_won = obj['AfterPotWon']

    def __str__(self):
        return '({0},{1},{2})'.format(self.after_player_action,self.after_board_dealed,self.after_pot_won)
